src/app/screens/Receive/index.tsx

```diff
@@ -232,47 +234,121 @@ function Receive() {
           />
         }
       />
-      {invoice ? (
-        <Container maxWidth="sm">{renderInvoice()}</Container>
-      ) : (
-        <form onSubmit={handleSubmit} className="h-full">
-          <fieldset className="h-full" disabled={loading}>
-            <Container justifyBetween maxWidth="sm">
-              <div className="py-4">
-                <div className="mb-4">
-                  <DualCurrencyField
-                    id="amount"
-                    min={0}
-                    label={t("amount.label")}
-                    placeholder={t("amount.placeholder")}
-                    fiatValue={fiatAmount}
-                    onChange={handleChange}
-                    autoFocus
-                  />
-                </div>
 
+      {isAlbyUser ? (
```

Comment:
In my opinion it would be nice to split these into separate components - ReceiveTypeTabs, LightningReceiveTab and OnchainReceiveTab.

Reply:
So if the user is an alby user then it should show LightningReciveTab and OnChainRecieveTab else it should show RecieveTypeTabs

Reply:
@HARI-124 RecieveTypeTabs would be the TabGroup which only the Alby user sees. for non-Alby connectors, I think we only need to show the content of the LightningReceiveTab. This should also fix the account switching bug I mentioned.